### PR TITLE
feat: add floating-vue, update VCopyright and add VCopyrightButton

### PR DESCRIPTION
**Pull request by Illawind**

### CODE REVIEW

File: components/atoms/VCopyrightButton/VCopyrightButton.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            name="copyright"
+            height="7"
+            width="7"
+        />
+    </VButton>
+</template>
+
+<style lang="scss" module>
+.root {
+    width: var(--v-information-button-width, #{rem(18 + 8 * 2)}); // Increase size for clickable zone on mobile
```
Comment (manuelodelain) [inline components/atoms/VCopyrightButton/VCopyrightButton.vue:18]: rename `--v-information-button-*` to `--v-copyright-button-*`

File: components/atoms/VCopyrightButton/VCopyrightButton.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    height: var(--v-information-button-height, #{rem(18 + 8 * 2)}); // Increase size for clickable zone on mobile
+    color: var(--v-information-button-foreground-color, #000);
+    isolation: isolate;
```
Comment (manuelodelain) [inline components/atoms/VCopyrightButton/VCopyrightButton.vue:21]: Why using `isolation` here?

File: components/molecules/VCopyright/VCopyright.vue [hunk 3]
```diff
@@ -1,79 +1,81 @@
...
-        :popper-triggers="['hover']"
-        :overflow-padding="16"
-        :aria-id="id"
-        :delay="{ show: 0, hide: 100 }"
+    <component
+        :is="displayedComponent"
         :class="$style.root"
+        v-bind="isTooltipComponentDisplayed ? tooltipProps : undefined"
+        @mouseover="onUserInteract"
+        @click="onUserInteract"
```
Comment (manuelodelain) [inline components/molecules/VCopyright/VCopyright.vue:53]: Does it work with focus?

File: package.json [hunk 4]
```diff
@@ -47,13 +47,15 @@
         "vite-svg-loader": "^5.1.0"
     },
     "dependencies": {
+        "@floating-ui/vue": "^1.1.5",
```
Comment (manuelodelain) [inline package.json:50]: needed? It doesn't seem to be used

File: components/molecules/VCopyright/VCopyright.vue [hunk 5]
```diff
@@ -1,79 +1,81 @@
...
+    }
+})
+
+const displayedComponent = shallowRef<string | typeof Tooltip>('div')
+const isTooltipComponentDisplayed = computed(() => displayedComponent.value !== 'div')
+
+async function onUserInteract() {
+    if (isTooltipComponentDisplayed.value) return
+
+    import('assets/scss/vendors/_floating-vue.scss')
```
Comment (manuelodelain) [inline components/molecules/VCopyright/VCopyright.vue:42]: why importing scss file on runtime? The css file needed is included into the CSS vendors. Should we import the file here?

**Review by manuelodelain — CHANGES_REQUESTED** [review]

File: components/molecules/VCopyright/VCopyright.vue (hunk 5 above)
Comment (Illawind) [reply components/molecules/VCopyright/VCopyright.vue:42]: the scss file will be compilated in a css file on runtime

**Review by manuelodelain — APPROVED** [review]

**Pull request merged**